tests/Doctrine/Tests/Models/Cache/City.php

```diff
@@ -9,25 +9,37 @@
 use Doctrine\ORM\Mapping as ORM;
 use Doctrine\ORM\Mapping\ClassMetadata;
 
-#[ORM\Entity, ORM\Table(name: 'cache_city'), ORM\Cache]
+#[ORM\Entity]
+#[ORM\Table(name: 'cache_city')]
+#[ORM\Cache]
 class City
 {
     /** @var int */
-    #[ORM\Id, ORM\GeneratedValue, ORM\Column(type: 'integer')]
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column(type: 'integer')]
     protected $id;
 
     /** @var Collection<int, Travel> */
     #[ORM\ManyToMany(targetEntity: 'Travel', mappedBy: 'visitedCities')]
     public $travels;
 
     /** @psalm-var Collection<int, Attraction> */
-    #[ORM\Cache, ORM\OrderBy(['name' => 'ASC'])]
+    #[ORM\Cache]
+    #[ORM\OrderBy(['name' => 'ASC'])]
     #[ORM\OneToMany(targetEntity: 'Attraction', mappedBy: 'city')]
     public $attractions;
 
     public function __construct(
-        #[ORM\Column(unique: true)] protected string $name,
-        #[ORM\Cache] #[ORM\ManyToOne(targetEntity: 'State', inversedBy: 'cities')] #[ORM\JoinColumn(name: 'state_id', referencedColumnName: 'id')] protected State|null $state = null,
+        #[ORM\Column(unique: true)]
+        protected string $name,
+        #[ORM\Cache]
+        #[ORM\ManyToOne(targetEntity: 'State', inversedBy: 'cities')]
+        #[ORM\JoinColumn(name: 'state_id', referencedColumnName: 'id')]
+        #[ORM\Cache]
+        #[ORM\ManyToOne(targetEntity: 'State', inversedBy: 'cities')]
+        #[ORM\Cache]
```

Comment:
Something weird happened here.

Reply:
I think it might be a bug in Slevomat's sniff fixer where multiple sniffs collided somehow (cc @kukulich). Is it blocking for us?

Reply:
👀

Reply:
> Is it blocking for us?

I think, I'd like to see this resolved before we release the new version. Otherwise I would need to review quite a lot of files and watch out for this kind of error.

Reply:
Looks good, thanks a lot!